edi_storage_oca/models/edi_backend.py

```diff
@@ -165,5 +167,9 @@ def _storage_get_input_filenames(self, exchange_type):
         pending_path_len = len(full_input_dir_pending)
         return [p[pending_path_len:].strip("/") for p in full_paths]
 
-    def _storage_new_exchange_record_vals(self, file_name):
-        return {"exchange_filename": file_name, "edi_exchange_state": "input_pending"}
+    def _storage_new_exchange_record_vals(self, file_name, storage_id):
```

Comment:
Rule of thumb: always try to not change the signature of an existing method w/ a required param.
In my opinion here is not useful anyway because this is an instance method and you can still use self.storage_id inside it. Do you see any problem w/ that?

Reply:
I updated it. Thanks @simahawk